.github/workflows/test.yaml

```diff
@@ -0,0 +1,60 @@
+name: Run tests
+
+on:
+  pull_request:
+    branches: [ master, main ]
+  push:
+    branches: [ master, main ]
+
+concurrency:
+  # cancel previous runs in current pull request
+  group: test-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 #3.5.3
+
+    - name: Setup PHP
+      uses: shivammathur/setup-php@7fdd3ece872ec7ec4c098ae5ab7637d5e0a96067 #2.26.0
+      with:
+        php-version: 8.2
+        tools: composer
```

Comment:
Please double-check this step. As far as I know, other PHP repos don't use Shivammathur anymore and I'm pretty sure it's not whitelisted (all non whitelisted actions will fail to run).

Reply:
You are right in that the other repositories don't seem to use it, looks like they instead run php in docker.. It's not fully clear to me why, but of course I can adjust it.

Reply:
I've replaced the setup-php action with a custom docker setup, similar to our other PHP repositories.